gpkit/geometric_program.py

```diff
@@ -152,7 +152,7 @@ def _get_solver(solver):
 
             if solver == "cvxopt":
                 from ._cvxopt import cvxoptimize_fn
-                solverfn = cvxoptimize_fn(*args, **kwargs)
+                solverfn = cvxoptimize_fn()
```

Comment:
`cvxoptimize_fn` did not accept args or kwargs, so this line fails if a user tries to specify args or kwargs.